src/components/Contribute/Knowledge/ReviewSubmission/index.tsx

```diff
@@ -2,114 +2,131 @@
 import { KnowledgeFormData } from '@/types';
 import { Content, ContentVariants } from '@patternfly/react-core';
 import React from 'react';
+import '../../Skill/ReviewSubmission/submission.css';
+import { Accordion, AccordionContent, AccordionItem, AccordionToggle } from '@patternfly/react-core';
 
 interface ReviewSubmissionProps {
   knowledgeFormData: KnowledgeFormData;
   isGithubMode: boolean;
 }
 
 export const ReviewSubmission: React.FC<ReviewSubmissionProps> = ({ knowledgeFormData, isGithubMode }) => {
+  const [expanded, setExpanded] = React.useState<{ [key: string]: boolean }>({});
+
+  const onToggle = (id: number) => {
+    setExpanded((prevState) => ({
+      ...prevState,
+      [id]: !prevState[id]
+    }));
+  };
+
   return (
-    <div>
-      <Content component={ContentVariants.h2}>Review Submission</Content>
+    <section className="review-submission-container">
+      <Content component={ContentVariants.h2}>Review</Content>
       {/* Author Information */}
-      <p>
-        <strong>Author Information</strong>
-      </p>
-      <p>
-        <i>Name:</i> {knowledgeFormData.name}
-      </p>
-      <p>
-        <i>Email:</i> {knowledgeFormData.email}
-      </p>
+      <article>
+        <div className="info-wrapper">
+          <p className="submission-titles">Author Information</p>
```

Comment:
To keep the same format across skill and knowledge review form, can you please update Author Information section to use "Contributor Information/Contributors" and add the relevant context text.